Deployf.Botf/System/UpdateContextExtensions.cs

```diff
@@ -104,6 +104,16 @@ public static string GetUserFullName(this IUpdateContext context)
 
         return first + " " + last;
     }
+    public static string GetLangCode(this IUpdateContext context)
+    {
+        var value = context.Update.Message?.From?.LanguageCode
+                    ?? context.Update.EditedMessage?.From?.LanguageCode
+                    ?? context.Update.CallbackQuery?.From?.LanguageCode
+                    ?? context.Update.InlineQuery?.From?.LanguageCode
+                    ?? "en";
+
+        return value;
```

Comment:
Why default value is english?
Default value must be null.

Reply:
English is default if other language not found